src/lib/isRgbColor.js

```diff
@@ -4,9 +4,18 @@ const rgbColor = /^rgb\((([0-9]|[1-9][0-9]|1[0-9][0-9]|2[0-4][0-9]|25[0-5]),){2}
 const rgbaColor = /^rgba\((([0-9]|[1-9][0-9]|1[0-9][0-9]|2[0-4][0-9]|25[0-5]),){3}(0?\.\d|1(\.0)?|0(\.0)?)\)$/;
 const rgbColorPercent = /^rgb\((([0-9]%|[1-9][0-9]%|100%),){2}([0-9]%|[1-9][0-9]%|100%)\)/;
 const rgbaColorPercent = /^rgba\((([0-9]%|[1-9][0-9]%|100%),){3}(0?\.\d|1(\.0)?|0(\.0)?)\)/;
+const startsWithRgb = /^rgba?/;
 
-export default function isRgbColor(str, includePercentValues = true) {
+export default function isRgbColor(str, includePercentValues = true, strict = true) {
```

Comment:
I would use an options object here instead of individual arguments. 

I think something should be done so that the function will be backwards compatible. There are examples of this somewhere in this repo, but I don't remember exactly where I've seen it

Reply:
We should indeed make `includePercentValues` backwards compatible if we move to an options object (which has my preference as well, see #1874 ). An example for this is;
https://github.com/validatorjs/validator.js/blob/86a07ba4f3f710f639e92a62cf81dd3321ef9ee8/src/lib/isLength.js#L8-L14

Reply:
is something 301e707 what you had in mind? :slightly_smiling_face:

Reply:
We don't want to present both ways of providing options in the readme. Just describe the object parameter. The backwards compatibility is just there to avoid breaking changes for existing codebases. I don't think we should actively promote the old (inferior) way.